pkg/scheduler/scheduler.go

```diff
@@ -252,6 +253,7 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 		}
 		log.Error(err, "Could not admit workload and assigning flavors in apiserver")
 		s.requeueAndUpdate(log, ctx, &e.Info, err.Error())
+		s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Pending", err.Error())
```

Comment:
The error by itself wouldn't tell much. Better to preface it with the same string as the log line in 254.